core/psh/Makefile

```diff
@@ -4,9 +4,9 @@
 # Copyright 2018, 2019, 2020, 2021 Phoenix Systems
 #
 
-OBJS = $(filter-out psh psh-empties, $(subst .c,,$(subst ./core/psh/,, $(shell find ./core/psh/ -name '*.c'))))
+OBJS = $(filter-out psh psh-empties login, $(subst .c,,$(subst ./core/psh/,, $(shell find ./core/psh/ -name '*.c'))))
```

Comment:
would be easier and less error-prone if applets would reside in a subdir.

I would advise to move the files when implementing registering them via constructors.